solr/core/src/test/org/apache/solr/cloud/api/collections/PurgeGraphTest.java

```diff
@@ -46,9 +46,7 @@ public void setUpRepo() throws Exception {
     repository = new LocalFileSystemRepository();
     baseLocationUri =
         repository.createDirectoryURI(
-            createTempDir("backup_files_" + UUID.randomUUID().toString())
-                .toAbsolutePath()
-                .toString());
+            createTempDir("backup_files_" + UUID.randomUUID()).toAbsolutePath().toString());
```

Comment:
Did you mean to remove the `toString` here?
```suggestion
            createTempDir("backup_files_" + UUID.randomUUID()).toAbsolutePath());
```